### PR TITLE
TWNTY-892 - Add ui/title stories

### LINKED ISSUE #892: Add ui/title stories
Add Storybook stories for components contained in the _modules/ui/title_ folder:
- MainSectionTitle
- SubSectionTitle

### PR DESCRIPTION
**Pull request by gitstart-twenty**
Fixes #892 

### Demo video
https://www.loom.com/share/0ce241baa1e94271b315bf99744647f3?sid=43b68165-3113-46fe-b476-4c8cb5fa9135

### CODE REVIEW

File: front/src/modules/ui/title/components/__stories__/MainSectionTitle.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+const args = {
+  children: 'Lorem ipsum',
+};
+
+export const Default: Story = {
+  args,
+  decorators: [ComponentDecorator],
+};
+
+export const Catalog: Story = {
```
Comment (charlesBochet) [inline front/src/modules/ui/title/components/__stories__/MainSectionTitle.stories.tsx:32]: as MainSectionTitle does not take anything beside {children} as a props, let's not add a catalog

File: front/src/modules/ui/title/components/__stories__/MainSectionTitle.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+  args,
+  argTypes: {
+    children: defaultArgTypes,
+  },
+  parameters: {
+    providers: Array(4).fill(ThemeProvider),
```
Comment (charlesBochet) [inline front/src/modules/ui/title/components/__stories__/MainSectionTitle.stories.tsx:38]: we should not do that, storybook is already aware of lightTheme and darkTheme that you can switch using storybook top bar

File: front/src/modules/ui/title/components/__stories__/MainSectionTitle.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+  title: 'UI/Title/MainSectionTitle',
+  component: MainSectionTitle,
+  decorators: [ComponentDecorator],
+};
+
+export default meta;
+
+type Story = StoryObj<typeof MainSectionTitle>;
+
+const defaultArgTypes = { control: false };
```
Comment (charlesBochet) [inline front/src/modules/ui/title/components/__stories__/MainSectionTitle.stories.tsx:21]: this should not be called defautArgTypes as this is used only for Catalog
As we are removing the catalog, let's remove this constant too

Comment (gitstart-twenty) [reply front/src/modules/ui/title/components/__stories__/MainSectionTitle.stories.tsx]: Hey @charlesBochet,
Change requests here have been addressed

**Review by charlesBochet — APPROVED** [review]
Thank you!

**Pull request merged**